### ISSUE TITLE
Build works but /bin/test does not

### ISSUE THREAD

**Issue by GonzaloSP** [metalsmith/metalsmith#240]
Hi there,

I am compiling a project with bin/build and works(Building website... Success), but when I execute bin/test I get:

>   features
>     subpage
>       1) adds .active css class to active menu item
> 
> 
>   0 passing (23ms)
>   1 failing
> 
>   1) features subpage adds .active css class to active menu item:
>      AssertionError: expected false to be true
>   

The website is pretty big I been trying to find the problem but no idea where it is, is there any log? or something I can try to narrow down where I should be looking for this?

Thanks

**Comment by woodyrew** [comment]
Metalsmith and a lot of the plugins use [debug](https://www.npmjs.com/package/debug).

TL;DR: Run with `DEBUG=*` before your build or test command.

**Comment by Ajedi32** [comment]
I don't really see how this has anything to do with Metalsmith. Neither `bin/build` nor `bin/test` part of Metalsmith. Presumably the site in question uses Metalsmith somehow (otherwise, why are you even posting this here?) but aside from that this doesn't sound like an issue with Metalsmith at all...
